### ISSUE TITLE
Non-Project support.

### ISSUE THREAD

**Issue by Carreau** [python3statement/python3statement.github.io#33]
I think there are organisation that might not have Python Project per se that might want to support us. 
Typically SWC that would provide tutorial that are Python 3 only maybe ? Should we add a different section ?

**Comment by takluyver** [comment]
We already have IAB, which is primarily learning materials rather than a package intended for other code to import. (pinging @jairideout)

If we get more of these, I'd be happy distinguishing them in some way. If it's just one or two, it's probably not worth it.

**Comment by jairideout** [comment]
@takluyver that's correct, IAB is an interactive bioinformatics text that uses Python 3 but the code there is for educational purposes only and not intended for import (we usually refer readers to scikit-bio for that).

👍 for differentiating if there's more projects added like this.

**Comment by Carreau** [comment]
Software carpentry might be interested as well, I started a private thread with Greg, which ended up as things will go in front of SWC steering committee later. 

I'm +1 on having a section for organisation in general. 

I think we should figure out how to do a second page with more FAQ like style.
